pkg/reference/parse.go

```diff
@@ -7,17 +7,14 @@ import (
 )
 
 var (
-	// ErrInvalid is used to return error if reference is invalid
-	ErrInvalid = errors.New("invalid reference")
-
 	// defaultTag is latest if there is no tag
 	defaultTag = "latest"
 )
 
 // Parse parses ref into reference.Named.
 func Parse(ref string) (Named, error) {
 	if ok := regRef.MatchString(ref); !ok {
-		return nil, ErrInvalid
+		return nil, errors.New("invalid reference: " + ref)
```

Comment:
The input from users, We don't need to do this maybe.

Reply:
Thanks for your kindly reminding~